README.md

```diff
@@ -16,12 +16,12 @@ An open source menu for N64 flashcarts.
 
 ## Current (notable) menu features
 * Fully Open Source.
-* Loads all known N64 games (including iQue and Aleck64 ROMs (even if they are byteswapped)).
+* Loads all known N64 games (including iQue and Aleck64 ROMs, even if they are byteswapped).
```

Comment:
This changes the context. Your suggested change implys that byteswapped ROM's are now only supported by Aleck and iQue.